lib/definition.js

```diff
@@ -174,25 +181,57 @@ const transform = (def, options) => {
         );
       return aliases.map(resolve);
     });
+
   /**
    * Resolve aliases inside prop values
    */
   const resolveAliases = def =>
     def.update("props", props => {
       const aliases = def.get("aliases", Immutable.Map());
-      return props.map((value, key) =>
-        value.update("value", v =>
-          recursiveMap(v, v => {
+
+      return props.map((value, key) => {
+        return value.update("value", v => {
+          return recursiveMap(v, v => {
             if (typeof v !== "string") return v;
             return allMatches(v, ALIAS_PATTERN).reduce((v, [alias, key]) => {
               if (!aliases.has(key))
                 throw new Error(`Alias "${key}" not found`);
               return v.replace(alias, aliases.getIn([key, "value"]));
             }, v);
-          })
-        )
-      );
+          });
+        });
+      });
+    });
+
+  /**
+   * Resolve aliases inside prop meta data
+   */
+  const resolveMetaAliases = def => {
+    return def.update("props", data => {
+      const aliases = def.get("aliases", Immutable.Map());
+
+      return data.map((value, key) => {
+        if (Immutable.Map.isMap(value.get("meta"))) {
```

Comment:
If there is no meta data, just pass through with original data. Do not attempt to resolve